resources.go

```diff
@@ -550,41 +548,12 @@ func Provider() tfbridge.ProviderInfo {
-			"azurerm_eventgrid_event_subscription":          {Tok: azureResource(azureMessaging, "EventGridEventSubscription")},
-			"azurerm_eventgrid_topic":                       {Tok: azureResource(azureMessaging, "EventGridTopic")},
-			"azurerm_eventhub":                              {Tok: azureResource(azureMessaging, "EventHub")},
-			"azurerm_eventhub_authorization_rule":           {Tok: azureResource(azureMessaging, "EventHubAuthorizationRule")},
-			"azurerm_eventhub_consumer_group":               {Tok: azureResource(azureMessaging, "EventHubConsumerGroup")},
-			"azurerm_eventhub_namespace":                    {Tok: azureResource(azureMessaging, "EventHubNamespace")},
-			"azurerm_eventhub_namespace_authorization_rule": {Tok: azureResource(azureMessaging, "EventHubNamespaceAuthorizationRule")},
-			"azurerm_servicebus_namespace": {
-				Tok: azureResource(azureMessaging, "Namespace"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					// https://docs.microsoft.com/en-us/rest/api/servicebus/create-namespace
-					// Max length of a servicehub namespace is 50.
-					azureName: AutoNameWithMaxLength(azureName, 50),
-				}},
-			"azurerm_servicebus_namespace_authorization_rule": {Tok: azureResource(azureMessaging, "NamespaceAuthorizationRule")},
-			"azurerm_servicebus_queue": {
-				Tok: azureResource(azureMessaging, "Queue"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					// https://groups.google.com/forum/#!topic/particularsoftware/XuHp_8wZ09o
-					// Max length of a servicehub queue is 260.
-					azureName: AutoNameWithMaxLength(azureName, 260),
-				}},
-			"azurerm_servicebus_queue_authorization_rule": {Tok: azureResource(azureMessaging, "QueueAuthorizationRule")},
-			"azurerm_servicebus_subscription":             {Tok: azureResource(azureMessaging, "Subscription")},
-			"azurerm_servicebus_subscription_rule":        {Tok: azureResource(azureMessaging, "SubscriptionRule")},
-			"azurerm_servicebus_topic": {
-				Tok: azureResource(azureMessaging, "Topic"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					// https://groups.google.com/forum/#!topic/particularsoftware/XuHp_8wZ09o
-					// Max length of a servicehub topic is 260.
-					azureName: AutoNameWithMaxLength(azureName, 260),
-				}},
-			"azurerm_servicebus_topic_authorization_rule": {Tok: azureResource(azureMessaging, "TopicAuthorizationRule")},
+			// EventHub
+			"azurerm_eventhub":                              {Tok: azureResource(azureEventHub, "EventHub")},
+			"azurerm_eventhub_authorization_rule":           {Tok: azureResource(azureEventHub, "EventHubAuthorizationRule")},
+			"azurerm_eventhub_consumer_group":               {Tok: azureResource(azureEventHub, "EventHubConsumerGroup")},
+			"azurerm_eventhub_namespace":                    {Tok: azureResource(azureEventHub, "EventHubNamespace")},
+			"azurerm_eventhub_namespace_authorization_rule": {Tok: azureResource(azureEventHub, "EventHubNamespaceAuthorizationRule")},
```

Comment:
Now when we have a proper module name, do we still want to prefix every resource with `EventHub`?

Reply:
Good idea! I can change these

Reply:
agreed.  Can we strip this before going in.

Reply:
done!